org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLFormattingOptions.java

```diff
@@ -29,6 +29,7 @@
 	public static final boolean DEFAULT_PRESERVE_ATTR_LINE_BREAKS = false;
 	public static final boolean DEFAULT_TRIM_TRAILING_SPACES = false;
 	public static final int DEFAULT_SPLIT_ATTRIBUTES_INDENT_SIZE = 2;
+	public static final boolean DEFAULT_CLOSING_TAG_NEW_LINE = false;
```

Comment:
In this file, there is a function called `initializeDefaultSettings()` where you should also set `closingTagNewLine` to its default value.